### ISSUE TITLE
False-positive E303 when using xml Module with xpath

### ISSUE THREAD

**Issue by maunzCache** [tsukinowasha/ansible-lint-rules#12]
Edit: Sorry was talking about the wrong thing.

If you use the Ansible xml module you may write xml content in any task definition using the xml module.

When combining it in a proper Ansible Task Syntax we'd produce
```
- name: "Some XPath Task"
  xml:
    path: "my-file.xml"
    xpath: "/x:server/x:profile/y:subsystem/y:datasources"
    input_type: xml
    set_children:
        - '<node attrA="a" attrB="b" attrC="c" attrD="d"'></node>
```

Obviously the E303 rule will now match the xml attributes because it is not considered in the context of the xml Module.
